omc3/definitions/optics.py

```diff
@@ -195,6 +195,7 @@ def delta_label(self):
 NORM_DISPERSION_COLUMN = ColumnsAndLabels(NORM_DISPERSION, _label=ylabels['norm_dispersion'], _text_label='normalized dispersion')
 PHASE_COLUMN =           ColumnsAndLabels(PHASE, _label=ylabels['phase'], _text_label='phase')
 TOTAL_PHASE_COLUMN =     ColumnsAndLabels(PHASE, _label=ylabels['phase'], _text_label='total phase')
+MU_COLUMN =              ColumnsAndLabels(PHASE_ADV, _label=ylabels['phasetot'], _text_label='total phase')
```

Comment:
This seems like exactly the line above considering that `ylabels['phase']` and `ylabels['phasetot']` are the same.

Could we just use the `TOTAL_PHASE_COLUMN` directly instead of adding a redundancy?

Reply:
No because PHASE is 'PHASE' and PHASE_ADV is 'MU'. This is causing issues when getting total phase correction predictions. The issue is that we treat phase and totalphase the same in our omc3/bbsrc results files, but they are different in models. One is detltaMu and the other is Mu

Reply:
Ah yes, I had not noticed that part! Good catch then